src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -161,11 +163,21 @@ doReadRow d k = forModuleNameFix $ \mnFix ->
         -- First, check: did we create this table during this block? If so,
         -- there's no point in looking up the key.
         checkDbTableExists tableName
-        result <- lift $ callDb "doReadRow"
-                       $ \db -> qry db queryStmt [SText rowkey] [RBlob]
+        (result,l) <- lift $ callDb "doReadRow"
+                       $ \db -> do
+          (t0 :: Time Micros) <- getCurrentTimeIntegral
```

Comment:
There exist a utility to generate traces telemetry message. However, we use that only conservatively and probably don't want to merge these logs here to master.

We could make them debug logs or hide them behind CPP. Or maybe a cost center pragma would be sufficient?

Reply:
I left this out for a later PR